docs-content/using/rotate-credentials.html.md.erb

```diff
@@ -0,0 +1,53 @@
+#### Application Host Identities
+
+When using application host identities (**Enable Spaced-scoped App Identities** is not checked), the host API
+key may be rotated by simply re-binding the application to the Conjur service:
+
+```sh
+cf unbind-service <app-name> conjur
+cf bind-service <app-name> conjur
+```
+
+> **NOTE:** This issues a new identity to the application, not just a new API key,
+> so any application specific permissions will need to be granted again. Any permissions
+> granted to the org or space layers is automatically inherited by the new identity.
+
+#### Space Host Identities
+
+Rotating the host API key when Space Host Identities are enabled (**Enable Spaced-scoped App Identities** is checked)
+requires coordinating the host credential update with all apps in a space.
+
+1. Rotate the Space Host API key using the Conjur CLI. This will return the new API key for the host:
```

Comment:
```suggestion
1. Rotate the Space Host API key using the Conjur CLI. This will return the new API key for the host:
```
```suggestion
1. Rotate the Space Host API key using the Conjur CLI. This returns the new API key for the host:
```